doc/release/release_dev.rst

```diff
@@ -208,53 +227,94 @@ A total of 217 changes have been committed.
 - PR for issue #6033 Improve test coverage for algorithms in betweenness_subset.py #6033 (#6083)
 - Di graph edges doc fix (#6108)
 - Improve coverage for core.py (#6116)
-- Add clear_edges() method to the list of methods to be frozen by nx.freeze() (#6190)
+- Add clear edges method to the list of methods to be frozen by the nx.… (#6190)
 - Adds LCA test case for self-ancestors from gh-4458. (#6218)
 - Minor Python 2 cleanup (#6219)
 - Add example laplacian matrix  (#6168)
 - Revert 6219 and delete comment. (#6222)
 - fix wording in error message (#6228)
 - Rm incorrect test case for connected edge swap (#6223)
-
+- add missing `seed` to function called by `connected_double_edge_swap` (#6231)
+- Hide edges with a weight of None in A*. (#5945)
+- Add dfs_labeled_edges reporting of reverse edges due to depth_limit. (#6240)
+- Warn users about duplicate nodes in generator function input (#6237)
+- Reenable geospatial examples (#6252)
+- Draft 3.0 release notes (#6232)
+- Add 2.8.x release notes (#6255)
+- doc: clarify allowed `alpha` when using nx.draw_networkx_edges (#6254)
+- Add a contributor (#6256)
+- Allow MultiDiGraphs for LCA (#6234)
+- Update simple_paths.py to improve readability of the BFS. (#6273)
+- doc: update documentation when providing an iterator over current graph to add/remove_edges_from. (#6268)
+- Fix bug vf2pp is isomorphic issue 6257 (#6270)
+- Improve test coverage for Eigenvector centrality  (#6227)
+- Bug fix in swap: directed_edge_swap and double_edge_swap  (#6149)
+- Adding a test to verify that a NetworkXError is raised when calling n… (#6265)
+- Pin to sphinx 5.2.3 (#6277)
+- Update pre-commit hooks (#6278)
+- Update GH actions (#6280)
+- Fix links in release notes (#6281)
+- bug fix in smallworld.py: random_reference and lattice_reference (#6151)
+- [DOC] Follow numpydoc standard in barbell_graph documentation (#6286)
+- Update simple_paths.py: consistent behaviour for `is_simple_path` when path contains nodes not in the graph. (#6272)
+- Correctly point towards 2.8.8 in release notes (#6298)
+- Isomorphism improve documentation (#6295)
+- Improvements and test coverage for `line.py` (#6215)
+- Fix typo in Katz centrality comment (#6310)
+- Broken link in isomorphism documentation (#6296)
+- Update copyright years to 2023 (#6322)
+- fix warnings for make doctest (#6323)
+- fix whitespace issue in 'networkx.generators.tests.test_internet_as_g… (#6324)
```

Comment:
This is a similar ellipsis issue, but the title of the PR had already been changed... unfortunately after the commit was merged...  So I guess changing the title of the PR after merging does not affect the git log. And that makes sense...  But it means that the script will not produce either of the PR titles I'm suggesting we use in the release notes.
```suggestion
- fix whitespace issue in test_internet_as_graph (#6324)
```